cloudinfo/resourcemanager.go

```diff
@@ -22,3 +25,40 @@ func (infoSvc *CloudInfoService) GetResourceGroupIDByName(resourceGroupName stri
 
 	return "", fmt.Errorf("resource group with name %s not found", resourceGroupName)
 }
+
```

Comment:
add comment with the function description so we get a popup in IDE and the golang docs will get updated random example https://github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper/blob/main/cloudinfo/catalog.go#L20